cmd/juju/commands/upgradejuju.go

```diff
@@ -56,8 +56,8 @@ used to allow the upgrade to proceed.
 Backups are recommended prior to upgrading.
 
 Examples:
-    juju upgrade-juju --dry-run
-    juju upgrade-juju --agent-version 2.0.1
+    juju upgrade-model --dry-run
+    juju upgrade-model --agent-version 2.0.1
```

Comment:
Should we rename the file "upgradejuju.go" to "upgrademodel.go" so developers can find it more easily?